localization/french/custom_localization/wc_insult_custom_loc_l_french.yml

```diff
@@ -1,9 +1,46 @@
 ﻿l_french:
  #InsultNoun
  rotting_corpse:0 "corps putride"
+ being_demon_insult_noun:0 "rejeton du Néant"
+ being_void_insult_noun:0 "abomination"
+ being_life_insult_noun:0 "forniqueur de cerfs"
+ being_order_insult_noun:0 "junkie"
```

Comment:
Not a fan of keeping "junkie" in French as it does not fit with the fantasy universe of wow, you might want to change it for something like "addict" or "accro"

Reply:
i added "addict" as you said and fix "forniqueur" as "fornicateur", also, i added customLoc, so those nouns are dynamics (Fornicatrice if female, fornicateur if male, etc etc)